Donation-platform/src/main/java/com/google/PersonalizedRecommendations.java

```diff
@@ -99,14 +110,19 @@
 
   // Gets charities from the database
   private Collection<Charity> getAllCharities() {
-    Collection<Charity> charities = db.getAllCharities();
+    Collection<Charity> charities = new ArrayList<>();
+    try {
+      charities = db.getAllCharities();
+    } catch (Exception e) {
+      System.out.println("Failure in retrieving charities: " + e);
```

Comment:
handle exception properly and not just print it

Reply:
@charu0291 I changed it to refer to the EntityNotFoundException explicitly. I will better handle these exceptions in the next PR, I am merging this one now so that we can get the master branch functional today.